### PR TITLE
Adapt reliability test to run with python 3.7

### PR DESCRIPTION
**Pull request by GGP1**
|Related issue|
|-------------|
|    Closes https://github.com/wazuh/wazuh-qa/issues/4299         |

## Description

Removed the walrus operator from the `tests/reliability/test_cluster/test_cluster_logs/test_cluster_master_logs_order/test_cluster_master_logs_order.py` test to support running it in Python 3.7.

> Re-created PR to merge it in **4.5.3**

### Logs

```console
(venv3.7) gasti@pop-os:~/work/wazuh-qa$ python --version
Python 3.7.17
(venv3.7) gasti@pop-os:~/work/wazuh-qa$ python3 -m pytest -vv --disable-warnings tests/reliability/test_cluster/test_cluster_logs/test_cluster_master_logs_order/ --artifacts_path=/home/gasti/work/wazuh-qa/tests/reliability/test_cluster/test_cluster_logs/test_cluster_master_logs_order --html=report.html --self-contained-html
=================================================================== test session starts ===================================================================
platform linux -- Python 3.7.17, pytest-7.1.2, pluggy-0.13.1 -- /home/gasti/work/wazuh-qa/venv3.7/bin/python3
cachedir: .pytest_cache
metadata: {'Python': '3.7.17', 'Platform': 'Linux-6.2.0-76060200-generic-x86_64-with-Pop-22.04-jammy', 'Packages': {'pytest': '7.1.2', 'pluggy': '0.13.1'}, 'Plugins': {'html': '3.1.1', 'testinfra': '5.0.0', 'metadata': '3.0.0'}}
rootdir: /home/gasti/work/wazuh-qa/tests, configfile: pytest.ini
plugins: html-3.1.1, testinfra-5.0.0, metadata-3.0.0
collected 1 item                                                                                                                                          

tests/reliability/test_cluster/test_cluster_logs/test_cluster_master_logs_order/test_cluster_master_logs_order.py::test_check_logs_order_master PASSED [100%]

-------------------------------------------- generated html file: file:///home/gasti/work/wazuh-qa/report.html --------------------------------------------
==================================================================== 1 passed in 0.01s ====================================================================
```

### Report

cluster_logs_Test

### CODE REVIEW

**Review by Selutario — APPROVED** [review]

**Review by Selutario — CHANGES_REQUESTED** [review]
@GGP1, I forgot to mention, please add the output of 3 executions (as [in this example](https://github.com/wazuh/wazuh-qa/pull/4313#pullrequestreview-1595569629))

**Comment by GGP1** [comment]
## Executions

<details><summary>Run 1</summary>

```console
(venv3.7) gasti@pop-os:~/work/wazuh-qa$ date; python3 -m pytest -vv --disable-warnings tests/reliability/test_cluster/test_cluster_logs/test_cluster_master_logs_order/test_cluster_master_logs_order.py --artifacts_path=artifacts --html=report.html --self-contained-html
Wed Aug 30 11:13:25 AM -03 2023
======================================================== test session starts =========================================================
platform linux -- Python 3.7.17, pytest-7.1.2, pluggy-0.13.1 -- /home/gasti/work/wazuh-qa/venv3.7/bin/python3
cachedir: .pytest_cache
metadata: {'Python': '3.7.17', 'Platform': 'Linux-6.2.0-76060200-generic-x86_64-with-Pop-22.04-jammy', 'Packages': {'pytest': '7.1.2', 'pluggy': '0.13.1'}, 'Plugins': {'html': '3.1.1', 'testinfra': '5.0.0', 'metadata': '3.0.0'}}
rootdir: /home/gasti/work/wazuh-qa/tests, configfile: pytest.ini
plugins: html-3.1.1, testinfra-5.0.0, metadata-3.0.0
collected 1 item                                                                                                                     

tests/reliability/test_cluster/test_cluster_logs/test_cluster_master_logs_order/test_cluster_master_logs_order.py::test_check_logs_order_master PASSED [100%]

--------------------------------- generated html file: file:///home/gasti/work/wazuh-qa/report.html ----------------------------------
========================================================= 1 passed in 0.01s ==========================================================
```

</details>


<details><summary>Run 2</summary>

```console
(venv3.7) gasti@pop-os:~/work/wazuh-qa$ date; python3 -m pytest -vv --disable-warnings tests/reliability/test_cluster/test_cluster_logs/test_cluster_master_logs_order/test_cluster_master_logs_order.py --artifacts_path=artifacts --html=report.html --self-contained-html
Wed Aug 30 11:13:46 AM -03 2023
======================================================== test session starts =========================================================
platform linux -- Python 3.7.17, pytest-7.1.2, pluggy-0.13.1 -- /home/gasti/work/wazuh-qa/venv3.7/bin/python3
cachedir: .pytest_cache
metadata: {'Python': '3.7.17', 'Platform': 'Linux-6.2.0-76060200-generic-x86_64-with-Pop-22.04-jammy', 'Packages': {'pytest': '7.1.2', 'pluggy': '0.13.1'}, 'Plugins': {'html': '3.1.1', 'testinfra': '5.0.0', 'metadata': '3.0.0'}}
rootdir: /home/gasti/work/wazuh-qa/tests, configfile: pytest.ini
plugins: html-3.1.1, testinfra-5.0.0, metadata-3.0.0
collected 1 item                                                                                                                     

tests/reliability/test_cluster/test_cluster_logs/test_cluster_master_logs_order/test_cluster_master_logs_order.py::test_check_logs_order_master PASSED [100%]

--------------------------------- generated html file: file:///home/gasti/work/wazuh-qa/report.html ----------------------------------
========================================================= 1 passed in 0.01s ==========================================================
```

</details>

<details><summary>Run 3</summary>

```console
(venv3.7) gasti@pop-os:~/work/wazuh-qa$ date; python3 -m pytest -vv --disable-warnings tests/reliability/test_cluster/test_cluster_logs/test_cluster_master_logs_order/test_cluster_master_logs_order.py --artifacts_path=artifacts --html=report.html --self-contained-html
Wed Aug 30 11:14:00 AM -03 2023
======================================================== test session starts =========================================================
platform linux -- Python 3.7.17, pytest-7.1.2, pluggy-0.13.1 -- /home/gasti/work/wazuh-qa/venv3.7/bin/python3
cachedir: .pytest_cache
metadata: {'Python': '3.7.17', 'Platform': 'Linux-6.2.0-76060200-generic-x86_64-with-Pop-22.04-jammy', 'Packages': {'pytest': '7.1.2', 'pluggy': '0.13.1'}, 'Plugins': {'html': '3.1.1', 'testinfra': '5.0.0', 'metadata': '3.0.0'}}
rootdir: /home/gasti/work/wazuh-qa/tests, configfile: pytest.ini
plugins: html-3.1.1, testinfra-5.0.0, metadata-3.0.0
collected 1 item                                                                                                                     

tests/reliability/test_cluster/test_cluster_logs/test_cluster_master_logs_order/test_cluster_master_logs_order.py::test_check_logs_order_master PASSED [100%]

--------------------------------- generated html file: file:///home/gasti/work/wazuh-qa/report.html ----------------------------------
========================================================= 1 passed in 0.01s ==========================================================
```

</details>

**Comment by GGP1** [comment]
## Executions

<details><summary>Run 1</summary>

```console
(venv3.7) gasti@pop-os:~/work/wazuh-qa$ date --utc; python3 -m pytest -vv --disable-warnings tests/reliability/test_cluster/test_cluster_logs/test_cluster_master_logs_order/test_cluster_master_logs_order.py --artifacts_path=artifacts --html=report.html --self-contained-html
Wed Aug 30 02:19:31 PM UTC 2023
======================================================== test session starts =========================================================
platform linux -- Python 3.7.17, pytest-7.1.2, pluggy-0.13.1 -- /home/gasti/work/wazuh-qa/venv3.7/bin/python3
cachedir: .pytest_cache
metadata: {'Python': '3.7.17', 'Platform': 'Linux-6.2.0-76060200-generic-x86_64-with-Pop-22.04-jammy', 'Packages': {'pytest': '7.1.2', 'pluggy': '0.13.1'}, 'Plugins': {'html': '3.1.1', 'testinfra': '5.0.0', 'metadata': '3.0.0'}}
rootdir: /home/gasti/work/wazuh-qa/tests, configfile: pytest.ini
plugins: html-3.1.1, testinfra-5.0.0, metadata-3.0.0
collected 1 item                                                                                                                     

tests/reliability/test_cluster/test_cluster_logs/test_cluster_master_logs_order/test_cluster_master_logs_order.py::test_check_logs_order_master PASSED [100%]

--------------------------------- generated html file: file:///home/gasti/work/wazuh-qa/report.html ----------------------------------
========================================================= 1 passed in 0.01s ==========================================================
```

</details>


<details><summary>Run 2</summary>

```console
(venv3.7) gasti@pop-os:~/work/wazuh-qa$ date --utc; python3 -m pytest -vv --disable-warnings tests/reliability/test_cluster/test_cluster_logs/test_cluster_master_logs_order/test_cluster_master_logs_order.py --artifacts_path=artifacts --html=report.html --self-contained-html
Wed Aug 30 02:19:42 PM UTC 2023
======================================================== test session starts =========================================================
platform linux -- Python 3.7.17, pytest-7.1.2, pluggy-0.13.1 -- /home/gasti/work/wazuh-qa/venv3.7/bin/python3
cachedir: .pytest_cache
metadata: {'Python': '3.7.17', 'Platform': 'Linux-6.2.0-76060200-generic-x86_64-with-Pop-22.04-jammy', 'Packages': {'pytest': '7.1.2', 'pluggy': '0.13.1'}, 'Plugins': {'html': '3.1.1', 'testinfra': '5.0.0', 'metadata': '3.0.0'}}
rootdir: /home/gasti/work/wazuh-qa/tests, configfile: pytest.ini
plugins: html-3.1.1, testinfra-5.0.0, metadata-3.0.0
collected 1 item                                                                                                                     

tests/reliability/test_cluster/test_cluster_logs/test_cluster_master_logs_order/test_cluster_master_logs_order.py::test_check_logs_order_master PASSED [100%]

--------------------------------- generated html file: file:///home/gasti/work/wazuh-qa/report.html ----------------------------------
========================================================= 1 passed in 0.01s ==========================================================
```

</details>

<details><summary>Run 3</summary>

```console
(venv3.7) gasti@pop-os:~/work/wazuh-qa$ date --utc; python3 -m pytest -vv --disable-warnings tests/reliability/test_cluster/test_cluster_logs/test_cluster_master_logs_order/test_cluster_master_logs_order.py --artifacts_path=artifacts --html=report.html --self-contained-html
Wed Aug 30 02:19:56 PM UTC 2023
======================================================== test session starts =========================================================
platform linux -- Python 3.7.17, pytest-7.1.2, pluggy-0.13.1 -- /home/gasti/work/wazuh-qa/venv3.7/bin/python3
cachedir: .pytest_cache
metadata: {'Python': '3.7.17', 'Platform': 'Linux-6.2.0-76060200-generic-x86_64-with-Pop-22.04-jammy', 'Packages': {'pytest': '7.1.2', 'pluggy': '0.13.1'}, 'Plugins': {'html': '3.1.1', 'testinfra': '5.0.0', 'metadata': '3.0.0'}}
rootdir: /home/gasti/work/wazuh-qa/tests, configfile: pytest.ini
plugins: html-3.1.1, testinfra-5.0.0, metadata-3.0.0
collected 1 item                                                                                                                     

tests/reliability/test_cluster/test_cluster_logs/test_cluster_master_logs_order/test_cluster_master_logs_order.py::test_check_logs_order_master PASSED [100%]

--------------------------------- generated html file: file:///home/gasti/work/wazuh-qa/report.html ----------------------------------
========================================================= 1 passed in 0.01s ==========================================================
```

</details>

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ All notable changes to this project will be documented in this file.
 Wazuh commit: TBD \
 Release report: TBD
 
+### Changed
+
+- Update the cluster master logs reliability test to run with python 3.7 [#4478](https://github.com/wazuh/wazuh-qa/pull/4478) \- (Tests)
```
Comment (Selutario) [inline CHANGELOG.md:12]:
```suggestion
- Update the cluster master logs reliability test to run with python 3.7 ([#4478](https://github.com/wazuh/wazuh-qa/pull/4478)) \- (Tests)
```

**Review by Selutario — APPROVED** [review]

**Review by damarisg — DISMISSED** [review]

**Pull request merged**